### PR TITLE
GLAM prod data without undesired metrics

### PR DESCRIPTION
**Pull request by edugfilho**
The idea is to have GLAM read from the views instead of `_v1` tables.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/glam_etl/glam_desktop_beta_aggregates/view.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-glam-prod-fca7.glam_etl.glam_desktop_beta_aggregates`
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/glam_etl/glam_desktop_beta_aggregates/view.sql:2]: I guess these should move into a `sql/moz-fx-data-glam-prod-fca7` folder

Comment (edugfilho) [reply sql/moz-fx-data-shared-prod/glam_etl/glam_desktop_beta_aggregates/view.sql]: Done!

**Review by scholtzan — APPROVED** [review]

**Comment by edugfilho** [comment]
@whd proactively asking for some guidance here: I'm creating these views so GLAM will read from them instead of the syndicate views on shared-prod. 
I moved them to glam-prod but I'm thinking they better be in `shared-prod`, since GLAM is reading from `shared-prod` already. Does that make sense? If so, can a syndicate dataset (shared-prod/glam_etl) contain views that are not of the syndicated project?

**Comment by edugfilho** [comment]
Despite my last comment, I decided to keep the views under glam-prod since shared-prod's glam_etl dataset will have syndicated views of them.

**Pull request merged**